src/m_menu.c

```diff
@@ -852,47 +849,50 @@ void M_DrawSaveLoadBottomLine(void)
 // M_LoadGame & Cie.
 //
 
-void M_DrawLoad(void)
+static void M_DrawLoad(void)
 {
-  int i;
+    int i;
 
-  //jff 3/15/98 use symbolic load position
-  V_DrawPatchDirect (72,LOADGRAPHIC_Y,W_CacheLumpName("M_LOADG",PU_CACHE));
-  for (i = 0 ; i < load_end ; i++)
+    //jff 3/15/98 use symbolic load position
+    V_DrawPatch(72, LOADGRAPHIC_Y, W_CacheLumpName("M_LOADG",PU_CACHE));
+    for (i = 0 ; i < load_end ; i++)
     {
-      M_DrawSaveLoadBorder(LoadDef.x,LoadDef.y+LINEHEIGHT*i);
-      M_WriteText(LoadDef.x,LoadDef.y+LINEHEIGHT*i,savegamestrings[i]);
+        menuitem_t *item = &currentMenu->menuitems[i];
+        char *cr = (item->flags & MF_HILITE) ? cr_bright : NULL;
```

Comment:
Here, for example, if you could replace `cr_bright` by `CR_HILITE` then we could change hilighting back to e.g. orange by redefining a single macro.

Reply:
Not quite, we don't have `CR_BRIGHT` yet. Is it worth adding it? Also we are currently using double translation (`V_DrawPatchTRTR` in `v_video.c`), maybe we should add more predefined tables, for example for `CR_GREEN_BRIGHT`?

Reply:
Pretty sure you already added it to `v_video.h`?

I'm fine with double translation tables, so we can keep color translations separate from shading or brightening translations and do not have to maintain (colors x shades) tables. Or do you see an advantage in the latter?

Reply:
> Pretty sure you already added it to `v_video.h`?

I didn't add it to `crdefs[]`, there is the code in `V_InitColorTranslation`:
```c
  cr_bright = malloc(256);
  for (int i = 0; i < 256; ++i)
  {
    cr_bright[i] = V_Colorize(playpal, CR_BRIGHT, (byte)i);
  }
```
So `colrngs[]` won't work. I thought `cr_bright` is a special case like `cr_dark`.

> I'm fine with double translation tables, so we can keep color translations separate from shading or brightening translations and do not have to maintain (colors x shades) tables. Or do you see an advantage in the latter?

Latter is slightly faster, but I agree, it's not worth it.

Reply:
> So `colrngs[]` won't work. I thought `cr_bright` is a special case like `cr_dark`.

Yes, it is a special case. Special enough to justify special treatment. 😉 

> Latter is slightly faster, but I agree, it's not worth it.

👍